lms/static/scripts/frontend_apps/components/SubmitGradeForm.js

```diff
@@ -46,7 +46,9 @@ const useFetchGrade = student => {
         const response = await fetchGrade({ student, authToken });
         if (!didCancel) {
           // Only set these values if we didn't cancel this request
-          setGrade(scaleGrade(response.currentScore, GRADE_MULTIPLIER));
+          if (response.currentScore) {
```

Comment:
note: because line 44 already pre-sets the grade to an empty string, we don't need an else condition